CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [12.3.0] - 2021-01-26
```

Comment:
Maybe update the link section at the bottom with
`[12.3.0]: https://github.com/lifeomic/cli/compare/v12.2.1...v12.3.0`

Reply:
ok